### PR TITLE
BZ-12743446 (6.3.x): fixing domain specific data pagination

### PR DESCRIPTION
**Pull request by Salaboy**
Fixed and tested in 6.3.x

### CODE REVIEW

**Comment by Salaboy** [comment]
I will close the one in master now.. because that need more work

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/test/list/ProcessInstanceListPresenterTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        presenter.setDataSetQueryHelperMock(dataSetQueryHelperMock);
+        presenter.setFilterSettingsMock(filterSettingsMock);
+        presenter.setDataSetQueryHelperDomainSpecificMock(dataSetQueryHelperDomainSpecificMock);
+        //Get the first page of process instances selected
+        Range range = new Range(0, 10);
+        
+        presenter.getData(range);
+     
+    }
+
```
Comment (csadilek) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/test/list/ProcessInstanceListPresenterTest.java]: @Salaboy this doesn't seem to be the latest version of your test though as this doesn't verify/assert anything. On master you had this: https://github.com/droolsjbpm/jbpm-console-ng/pull/205/files#diff-46695cca38a69565d252ef87a26c8b0b

Comment (csadilek) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/test/list/ProcessInstanceListPresenterTest.java]: @Salaboy ah I see, never mind, you moved this to the MyDataSetProcessInstanceWithVariablesListPresenter.

**Comment by csadilek** [comment]
Thanks!

6.3.x: c3e3dfad16732080be302d3290ee28bc1ab55c85

**Pull request closed without merging**